src/components/mortgage-investment-compare/index.js

```diff
@@ -174,12 +194,17 @@ class MortgageInvestmentCompare extends React.Component {
           <h2>Compare Mortgage Terms</h2>
           <p className='mt-4'>
             Is a {this.state.result.option1.term} year mortgage better than a{' '}
-            {this.state.result.option2.term} mortgage? See what may happen in
-            each situation, with the same budget .
+            {this.state.result.option2.term} year mortgage? See what happens in
+            each situation, with the same budget.
           </p>
           <InputCard
             onInputChange={this.updateInput}
+            onROISwitch={this.toggleIncludeROI}
+            onInflationSwitch={this.toggleIncludeInflation}
+            includeInflation={this.state.includeInflation}
+            includeROI={this.state.includeROI}
             input={this.state.input}
+            updtateResult={this.updtateResult}
```

Comment:
Why does the input card need to update the result directly?

Reply:
just like it updates the input, please see long answer below